client/graphql/queries/userProfileSets.graphql

```diff
@@ -0,0 +1,49 @@
+query userProfileSets(
+  $username: String!
+  $first: Int!
+  $after: String
+  $filters: CustomSetFilters!
+) {
+  userByName(username: $username) {
+    username
+    id
+    customSets(first: $first, after: $after, filters: $filters) {
+      edges {
+        node {
+          id
+          name
+          level
+          defaultClass {
+            id
+            name
+            faceImageUrl
+          }
+          equippedItems {
+            id
+            slot {
+              id
+              order
+            }
+            item {
+              id
+              imageUrl
+            }
+          }
+          tagAssociations {
+            id
+            associationDate
+            customSetTag {
+              id
+              name
+              imageUrl
+            }
+          }
+        }
```

Comment:
If this is exactly the same as the "myCustomSets" query, it would be good to put it into a fragment. Or even change that MyBuilds component to just use this query with the current user's username.